deepchem/utils/evaluate.py

```diff
@@ -128,3 +129,91 @@ def compute_model_performance(self,
       return multitask_scores
     else:
       return multitask_scores, all_task_scores
+
+
+class GeneratorEvaluator(object):
+  """Class that evaluates a model on a given dataset."""
```

Comment:
This needs better documentation too.  The description doesn't even make sense, given that no Dataset appears in its API.